active/InformationBar/InformationBar.md

```diff
@@ -0,0 +1,536 @@
+</StackPanel>
+```
+
+![A mockup of an InfoBar in the top of the content area. The notification's title is "New Photos From Cloud" and it's message is "Sync your latest photos to the Cloud". It has a bright green accent color and a "Camera" icon.](images/Custom_IconColor.png)
+
+## Add buttons
+By default, an 'X' close button will appear as the right most component in the bar.
+
+An additional action button can be added by setting the ActionButtonContent and ActionButtonCommand properties. There is also built-in support for a single hyperlink button to ensure text contrast remains accessible with the various background colors. We recommend that only a single action button or hyperlink button is set. Outside of these, additional action buttons and hyperlinks can be added via custom content.
+
+XAML
+```xml
+<StackPanel x:Name="ContentArea">
+    <controls:InfoBar x:Name="UnsuccessfulSaveNotification"
+        Severity="Warning"
+        Title="Error while saving"
+        Message="Your document was unable to be saved."
+        ActionButtonContent="Try again"
+        ActionButtonCommand="SaveDocuments">
+    </controls:InfoBar>
+</StackPanel>
+```
+
+![A mockup of an InfoBar with a single line message and an action button](images/Warning_Button.png)
+
+XAML
+```xml
+<StackPanel x:Name="ContentArea">
+    <controls:InfoBar x:Name="ConnectionErrorNotification"
+        Title="Error while saving"
+        Message="Lorem ipsum long message">
+        <controls:InfoBar.HyperlinkButtonContent>
+            <HyperlinkButton
+                Content="www.microsoft.com" 
+                NavigateUri="http://www.microsoft.com"/>
+        <controls:InfoBar.HyperlinkButtonContent/>
+    </controls:InfoBar>
+</StackPanel>
+```
+![A mockup of an InfoBar with a single line message and hyperlink](images/Default_MessageHyperlink.png)
+![A mockup of an InfoBar with a message expanding multiple lines and a hyperlink](images/Docked_MultiLineIconTitleHyperlink.png)
+
+## Custom content
+Content can be added to an InfoBar using the Content property. It will appear below the message and above the action or hyperlink buttons if defined. The InfoBar will expand to fit the content defined.
+
+XAML
+```xml
+<StackPanel x:Name="ContentArea">
+    <controls:InfoBar x:Name="TermsAndConditionsNotification"
+        Title="Update Complete!"  
+        Message="Before you can proceed you must read and accept the new Terms and Conditions.">
+        <controls:InfoBar.HyperlinkButtonContent>
+            <HyperlinkButton Content="Read here" NavigateUri="www.microsoft.com"/>
+        </controls:InfoBar.HyperlinkButtonContent>
+    </controls:InfoBar>
+</StackPanel>
+```
+
+![A sketch of an InfoBar in its default state with a hyperlink and message](images/Information_CustomContent.jpg)
+Mockup with custom content to be added.
+
+## Content wrapping
+By default, the text set in the Message property will wrap vertically in the control underneath the other visual components.
```

Comment:
Is there a way to override the default?

Reply:
Should there be? The alternative would be to allow the content to cut itself off once it is too long. That's an option that Fluent UI's [MessageBar](https://developer.microsoft.com/en-us/fluentui#/controls/web/messagebar) has in their isMultiline property but I struggle to see a scenario where we would want to support that.

Or are you more focused on the wording of "by default"?

Reply:
Yeah, it says "by default" indicating that there's a non-default case, but not explaining it.